addons/cephfs/readme.md

```diff
@@ -0,0 +1,88 @@
+# Ceph
+
+This addon is built based for [Ceph](https://ceph.com/).
+
+## Install
+
+```shell
+kubectl apply -f runtime-profile.yaml
+```
+
+## How to use
+
+### Create and Deploy Dataset and ThinRuntime Resource
+```shell
+$ cat <<EOF > dataset.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: ceph-demo
+spec:
+  mounts:
+  - mountPoint: ceph://<IP:Port>
+    name: ceph-pvc
+    options:
+      fsid: <fsid>
+      mon_initial_members: <mon_initial_members>
+      mon_host: <mon_host>
+      auth_cluster_required: <auth_cluster_required>
+      auth_service_required: <auth_service_required>
+      auth_client_required: <auth_client_required>
+      key: <key>
+---
+apiVersion: data.fluid.io/v1alpha1
+kind: ThinRuntime
+metadata:
+  name: ceph-demo
+spec:
+  profileName: ceph-profile
+EOF
+
+$ kubectl apply -f dataset.yaml
+```
+Modify the above mountPoint to the address of the remote Ceph you want to use, and fill  the verification information in the `options` field.
+
+### Run pod with Fluid PVC 
+
+```shell
+$ cat <<EOF > app.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+    - name: nginx
+      image: nginx
+      volumeMounts:
+        - mountPath: /data
+          name: nfs-demo
+  volumes:
+    - name: nfs-demo
```

Comment:
Why using the name `nfs` ?

Reply:
Oh, I just followed the nfs doc and forgot to change it. I will change it.